### DISCUSSION TITLE
形码纯单字模式中能否临时切换成纯词组模式

### DISCUSSION THREAD

**Discussion by CHANTXU64** [rime/home#1496] [Q&A] [answered]
能不能做到从纯单字模式快速临时切换成纯词组模式，比如按下分号键就切换到纯词组模式，完成这次输入或者退出输入就会切换回去纯单字。有没有作业抄，我找了好多方案，都没有，要么就是混合输入，要么就是完全分离的要control+`切换到菜单更换。

**Comment by gaboolic** [comment]
可以用两个词典
按下引导 用另一个词典
比如https://github.com/functoreality/rime-flypy-zrmfast  按下aw ：单词模式

**Comment by yanhuacuo** [comment]
形码都有【临时拼音】，仿写换表即可。

**Comment by CHANTXU64** [comment, marked as answer]
第一个回答，试了很久也没办法正常工作。第二个回答，这应该是RIME框架自带的拼音反查功能，简单试了下，确实也不行。

后来我试了直接在码表里改码，加个前缀字符就行，再稍微改下配置文件就行了。后面搞完了我再分享出来
